app/.meteor/packages

```diff
@@ -9,16 +9,15 @@ mobile-experience@1.0.4       # Packages for a great mobile UX
 mongo@1.1.12                   # The database Meteor supports right now
 blaze-html-templates@1.0.4    # Compile .html files into Meteor Blaze views
 reactive-var@1.0.10            # Reactive variable for tracker
-jquery@1.11.9                  # Helpful client-side library
+jquery                  # Helpful client-side library
```

Comment:
Versions should be specified if possible to keep dev environments and builds consistent.